src/CPUActivityAgent.cpp

```diff
@@ -487,16 +475,26 @@ namespace geopm
     {
         std::vector<std::pair<std::string, std::string> > result;
 
-        result.push_back({"Core Frequency Requests", std::to_string(m_core_frequency_requests)});
-        result.push_back({"Core Clipped Frequency Requests", std::to_string(m_core_frequency_clipped)});
-        result.push_back({"Uncore Frequency Requests", std::to_string(m_uncore_frequency_requests)});
-        result.push_back({"Uncore Clipped Frequency Requests", std::to_string(m_uncore_frequency_clipped)});
-        result.push_back({"Resolved Maximum Core Frequency", std::to_string(m_resolved_f_core_max)});
-        result.push_back({"Resolved Efficient Core Frequency", std::to_string(m_resolved_f_core_efficient)});
-        result.push_back({"Resolved Core Frequency Range", std::to_string(m_resolved_f_core_max - m_resolved_f_core_efficient)});
-        result.push_back({"Resolved Maximum Uncore Frequency", std::to_string(m_resolved_f_uncore_max)});
-        result.push_back({"Resolved Efficient Uncore Frequency", std::to_string(m_resolved_f_uncore_efficient)});
-        result.push_back({"Resolved Uncore Frequency Range", std::to_string(m_resolved_f_uncore_max - m_resolved_f_uncore_efficient)});
+        result.push_back({"Core Batch Writes",
+                          std::to_string(m_core_batch_writes)});
+        result.push_back({"Core Frequency Requests Clipped",
+                          std::to_string(m_freq_governor->get_clamp_count())});
```

Comment:
Make this "clamped" everywhere to align with what's in the FrequencyGovernor.  When we go to refactor these later, if we want to make it clipped, we can do it in all the spots.